docs/source/conf.py

```diff
@@ -19,23 +19,33 @@
 
 # -- Project information -----------------------------------------------------
 import os
+import re
 
 
-def get_version():
-    version_file_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-    with open(os.path.join(version_file_dir, 'VERSION')) as version_file:
-        ver = version_file.read().strip()
-    return ver
+docs_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
To be more specific and explicit 
```suggestion
docs_parent_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```